examples/push-script.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+#script takes first param as current Voyager release version and next params the names of directories of examples that should be pushed
+
+echo "Publishing version $1 of voyager-server examples to Docker"
+
+i=1
+for dir in "$@"
+do
+  if [ "$i" -gt 1 ]; then
+      echo "Building docker for example $dir"
+      DOCKERFILE="Dockerfile.$dir"
+      CONTAINER="aerogear/voyager-server-example-$dir:$1"
+      cat Dockerfile | sed -e "s/\$directory/$dir/g" > "$DOCKERFILE"
```

Comment:
This approach of using sed to replace the `$directory` string is kinda confusing. Dockerfile as support for something called build args which should probably be used instead. https://docs.docker.com/engine/reference/builder/#arg

Basically in the dockerfile you would make a declaration `ARG directory`

and then when you build you can do `docker build --build-arg directory=$DIR ...`

If you use a build arg, this script will become much simpler as you will not need to make copies of the Dockerfile for each example you are building 🙂

Reply:
Ok, my lack of knowledge. I'll fix it to do that...